docs/guides/operator-guide-aws-marketplace.md

```diff
@@ -252,6 +252,69 @@ Here's what you need to know about upgrading:
 
 >TIP: For a smooth transition, create the new stack first. After the new stack is bootstrapped and ready to serve, delete the old stack. If you are using a load balancer, first get the new instances up and running and then convert the DNS name from the previous one to the new one.
 
+## Logs
+### Where to read logs
+To access the logs, the participants need to ssh into the EC2 instance and access the logs.
+
+The logs are located at `/var/logs/` . The logs are in the format of `operator.log-<timestamp rotated>` . 
+
+### Default log settings
+#### logrotate config
+- After deploying the operator instance, the default log rotation settings will be applied:
+    - 30 rotations of logs will be kept
+    - Normally if no abnormal logs, it should be equivalent to 30 days' worth of logs as it will be rotated daily
+    - If abnormality is encountered and the log size increased abnormally, the log will be rotated once it reaches 30MB
```

Comment:
(also space between number and MB) Suggest:

- If log entries are very large, and the log size reaches 30 MB within a 24-hour period, the log is rotated at that point.